spark/src/main/java/org/datacleaner/spark/TransformationAction.java

```diff
@@ -0,0 +1,47 @@
+/**
+ * DataCleaner (community edition)
+ * Copyright (C) 2014 Neopost - Customer Information Management
+ *
+ * This copyrighted material is made available to anyone wishing to use, modify,
+ * copy, or redistribute it subject to the terms and conditions of the GNU
+ * Lesser General Public License, as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this distribution; if not, write to:
+ * Free Software Foundation, Inc.
+ * 51 Franklin Street, Fifth Floor
+ * Boston, MA  02110-1301  USA
+ */
+package org.datacleaner.spark;
+
+import java.util.List;
+
+import org.apache.spark.api.java.function.FlatMapFunction;
+import org.datacleaner.api.InputRow;
+import org.datacleaner.job.runner.ConsumeRowHandler;
+import org.datacleaner.job.runner.ConsumeRowResult;
+
+public final class TransformationAction extends AbstractSparkDataCleanerAction implements FlatMapFunction<InputRow, InputRow> {
+    
+    private static final long serialVersionUID = 1L;
+
+    public TransformationAction(final SparkDataCleanerContext sparkDataCleanerContext) {
+        super(sparkDataCleanerContext);
+    }
+
+    @Override
+    public List<InputRow> call(InputRow inputRow) throws Exception {
+        ConsumeRowHandler.Configuration configuration = new ConsumeRowHandler.Configuration();
+        configuration.includeAnalyzers = false;
+        ConsumeRowHandler consumeRowHandler = new ConsumeRowHandler(getAnalysisJob(), getDataCleanerConfiguration(),
+                configuration);
```

Comment:
Creating the handler is sorta heavy weight and each as instantiation will create new analyzers etc. This should be done only once per partition, so I think we should first call RDD.glom() and then build this handler for each action iterator in the glom's stream.